suite-native/assets/src/assetsSelectors.ts

```diff
@@ -45,25 +45,20 @@ export const selectBalancesPerNetwork = memoize((state: AssetsRootState): Format
 export const selectAssetsWithBalances = memoize((state: AssetsRootState) => {
     const balancesPerNetwork = selectBalancesPerNetwork(state);
     const networksWithAssets = Object.keys(balancesPerNetwork) as NetworkSymbol[];
-    const coins = selectCoinsLegacy(state);
 
-    const fiatCurrency = selectFiatCurrencyCode(state);
+    const fiatCurrencyCode = selectFiatCurrencyCode(state);
 
     return networksWithAssets
         .map((networkSymbol: NetworkSymbol) => {
             const network = networks[networkSymbol];
+            const fiatRate = selectFiatRatesByTicker(state, { symbol: networkSymbol });
```

Comment:
You don't need new selector, you can use this one:
```tsx
const currentFiatRates = selectFiatRatesByFiatRateKey(
                state,
                getFiatRateKey(networkSymbol, fiatCurrency),
            );
```

Reply:
Alright. I've created this selector on @juriczech request. I will revert this fixup then and keep the original solution with the selector you suggest.

Reply:
fixed in fixup fixup https://github.com/trezor/trezor-suite/commit/c2e42d090e921371cfd42ad5645b7d7eabb49430. Same as above.